github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -1881,6 +1881,42 @@ presubmits:
           privileged: true
       nodeSelector:
         type: bare-metal-external
+  - always_run: true
+    optional: true
+    annotations:
+      fork-per-release: "true"
+      testgrid-dashboards: kubevirt-presubmits
+    cluster: prow-workloads
+    decorate: true
+    decoration_config:
+      grace_period: 5m0s
+      timeout: 1h0m0s
+    labels:
+      preset-bazel-cache: "true"
+      preset-bazel-unnested: "true"
+      preset-dind-enabled: "true"
```

Comment:
I think this label is not needed for this check - it doesn't require docker in docker

Reply:
Since the lane doesn't atm run Go code either, I have removed the othe labels as well for now.
[Done](https://github.com/kubevirt/project-infra/compare/50af0d89e409b40261dddee10c90afe9299efe13..498d2712f9cba2c5ea9647c5cd468cd14f3b0ded)